tools/alive-tv.cpp

```diff
@@ -292,6 +292,11 @@ static void compareFunctions(llvm::Function &F1, llvm::Function &F2,
     return;
   }
   Func1->unroll(opt_src_unrolling_factor);
+  if (opt_src_unrolling_factor) {
+    cout << "\n----------------------------------------\n";
+    cout << "Unrolled source\n";
```

Comment:
you can remove 'cout <<' from this 2nd line to save some code size 😀
(same below)